environment.go

```diff
@@ -785,6 +793,13 @@ type BuildBaronProject struct {
 	BFSuggestionFeaturesURL string `mapstructure:"bf_suggestion_features_url" bson:"bf_suggestion_features_url"`
 }
 
+type AnnotationsSettings struct {
+	// a list of jira fields the user wants to display in addition to state assignee and priority
+	JiraCustomFields []string `mapstructure:"jira_custom_fields" bson:"jira_custom_fields"`
+	// the endpoint that the user would like to send data to when the file ticket button is clicked
+	WebHook string `mapstructure:"web_hook" bson:"web_hook"`
```

Comment:
maybe we should allow users to specify a secret that gets sent with the webhook, like [GitHub does](https://developer.github.com/webhooks/creating/#secret).
For user subscriptions we generate [a secret](https://github.com/evergreen-ci/evergreen/blob/05d7b78d23cd59007ec5378783ab5aa65fbb704b/public/static/js/subscriptions.js#L140).

Reply:
done